restcontrollers/common.go

```diff
@@ -0,0 +1,146 @@
+package restcontrollers
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/artofimagination/mysql-resources-db-go-service/dbcontrollers"
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+)
+
+type RESTController struct {
+	DBController *dbcontrollers.MYSQLController
+}
+
+const (
+	ResourceURIAdd                   = "/add-resource"
+	ResourceURIGetByID               = "/get-resource-by-id"
+	ResourceURIGetMultipleByIDs      = "/get-resources-by-ids"
+	ResourceURIGetMultipleByCategory = "/get-resources-by-category"
+	ResourceURIUpdate                = "/update-resource"
+	ResourceURIDelete                = "/delete-resource"
+	ResourceURIGetCategories         = "/get-categories"
+)
+
+const (
+	POST = "POST"
```

Comment:
You could use these constants from the http lib.
http.MethodPost and http.MethodGet (if I remember well)